Code/C++/palindrome_number.cpp

```diff
@@ -0,0 +1,56 @@
+/*       Palindrome Number : Given an integer x, return true if x is palindrome integer.
+                            An integer is a palindrome when it reads the same backward as forward.
+                            This code is implemented without using extra space.                       
```

Comment:
+ add algorithm too!

Reply:
@rish-singhal  it's just a mathematical type logic ,so it doesn't include any famous algo . Still I have added the description of the code .

Reply:
What are you doing to check if it is palindrome, add the steps.